experiments/benchmarking/dgps.R

```diff
@@ -89,7 +91,17 @@ gen_data <- function(n, p, sigma.m = 1, sigma.tau = 0.1, sigma.noise = 1,
     zeta1 <- 1 + 1 / (1 + exp(-20 * (X[, 1] - (1 / 3))))
     zeta2 <- 1 + 1 / (1 + exp(-20 * (X[, 2] - (1 / 3))))
     tau <- zeta1 * zeta2
-    e <- (1 / 4) * (1 + dbeta(X[, 1], 2, 4))
+    e <- (1 / 4) * (1 + dbeta(X[, 1], 2, 4))  # Confounding
+    W <- rbinom(n = n, size = 1, prob = e)
+    m <- 2 * X[, 1] - 1 + e * tau
+    V <- 1
```

Comment:
Just a tiny note: there is nothing wrong here, was just wondering why the diff algorithm shows this as as changed instead of the block just before "else if (dgp == "ai1")"